### ISSUE TITLE
Allow `loading` attribute for <img> tag

### ISSUE THREAD

**Issue by neluzhin** [ampproject/amphtml#22802]
When using `loading` attribute on `<img>` tag AMP validator fails with following message:

> The attribute 'loading' may not appear in tag 'img'. (see https://amp.dev/documentation/components/amp-img)

Code:

```html
<amp-img src="https://example.com/image.png" alt="Example" layout="fill">
  <noscript>
    <img src="https://example.com/image.png" alt="Example" loading="lazy">
  </noscript>
</amp-img>
```

I use `<noscript>` as fallback for users with disabled JavaScript and some kind of crawlers who can't execute `<amp-img>` element (as described in docs example: https://amp.dev/documentation/examples/components/amp-img/#no-script)

This issue may be relevant to #21982 and probably #21497.

**Comment by aghassemi** [comment]
`amp-img` itself does lazy loading but I agree with allowing `loading` for `noscript` case.
